packages/mdc-layout-grid/_mixins.scss

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+@import "./variables"
+
 // returns the lower grid boundary or null if the smallest grid is selected
```

Comment:
Looks like the linter is complaining here

`17:2  ✖  Expected indentation of 2 spaces   indentation`

Reply:
I don't think I get that. What should be fixed?

Reply:
You should indent the line 17 by 2 spaces. Not more, not less.

Reply:
That's surprising I thought it was correctly indented after seeing it. Moreover, I didn't touch that line at all in the first place.

Reply:
Maybe someone edited it in the past but it got accepted even with the error

Reply:
I can't take this. This is so odd. Why doesn't the linter warn about a missing indentation at [line 27](https://github.com/sivaraam/material-components-web/blob/15be7c54edf3d0a6d8b7e70cd18045997d92064a/packages/mdc-layout-grid/_mixins.scss#L27) or [line 39](https://github.com/sivaraam/material-components-web/blob/15be7c54edf3d0a6d8b7e70cd18045997d92064a/packages/mdc-layout-grid/_mixins.scss#L39) which has a similar comment to this line?

Reply:
Your text editor may have parsed and saved it incorrectly. You can always run `npm run lint` to see what will run on TravisCI.

Reply:
Ok, I found the issue. I missed a semicolon on line 15.